lib/govuk_tech_docs/api_reference/api_reference_renderer.rb

```diff
@@ -18,20 +18,15 @@ def initialize(app, document)
       end
 
       def api_full(info, servers)
-        paths = ''
-        paths_data = @document.paths
-        paths_data.each do |path_data|
-          # For some reason paths.each returns an array of arrays [title, object]
-          # instead of an array of objects
-          text = path_data[0]
-          paths += path(text)
+        paths = @document.paths.keys.inject('') do |memo, text|
```

Comment:
`#inject` pre-dates `#reduce` in Ruby so that seems the more idiomatic way to do it for me. I wouldn't suggest it needs changing here as we've never applied any preference in GDS code for one way or the other.